lib/3scale_toolbox/commands/service_command/service_show.rb

```diff
@@ -0,0 +1,99 @@
+module ThreeScaleToolbox
+  module Commands
+    module ServiceCommand
+      class ServiceShowSubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+
+        def self.command
+          Cri::Command.define do
+            name        'show'
+            usage       'show'
+            summary     'service show'
+            description "Show the information of a service"
+            runner ServiceShowSubcommand
+
+            param   :source
```

Comment:
instead of `source` (which makes sense on copy command), what about `remote`???

Reply:
I chose `source` because a URI can be used instead of a remote name

Reply:
`remote` for me it is either a remote name stored in toolbox config or URI. If it is confusing, we should state it clear in doc or cli help

Reply:
renamed to 'remote'